src/sinks/aws_kinesis_firehose/service.rs

```diff
@@ -41,10 +42,12 @@ impl Service<Vec<KinesisRequest>> for KinesisService {
     type Error = SdkError<PutRecordBatchError>;
     type Future = BoxFuture<'static, Result<Self::Response, Self::Error>>;
 
+    // Emission of Error internal event is handled upstream by the caller
     fn poll_ready(&mut self, _cx: &mut Context) -> Poll<Result<(), Self::Error>> {
         Poll::Ready(Ok(()))
     }
 
+    // Emission of Error internal event is handled upstream by the caller
```

Comment:
From discussion on the statsd sink PR , I removed these comments from the Service impls in the code, and instead placed it in the Trait function definition.
So assuming you have pulled that change into your branch, you should be able to see this information in the highlight hint for these functions when hovering.

```suggestion
    fn poll_ready(&mut self, _cx: &mut Context) -> Poll<Result<(), Self::Error>> {
        Poll::Ready(Ok(()))
    }
    
```

Reply:
Moved in https://github.com/vectordotdev/vector/pull/14478/commits/752f7f4eb565cfce9d74b8997cd622cf37cfbbee